### ISSUE TITLE
Guilds scope in DiscordUser type

### ISSUE THREAD

**Issue by harbar20** [alii/nextjs-discord-oauth#2]
Hi! Thank you for creating this, this is super helpful.

I want to add the "guilds" scope to the OAuth2 flow so I can use it in my project. However, I have no clue how I'd go about adding a "guilds" property to the `DiscordUser` type. 

Any suggestions/advice?

**Comment by harbar20** [comment]
Ah, I see #1 is the exact same. My issue is more of how to store `guilds` in a cookie at all. I could use context but I feel like splitting up `guilds` and `user` like this, when I want them to be linked, is bad. Storing `guilds` in a cookie is not possible because signing it creates a token that is longer than is allowed to be stored in a cookie.

Any advice to this end?

**Comment by harbar20** [comment]
Figured it out! Instead of storing the whole user object, I just stored the access token instead, setting the `expiresIn` property to be the `expires_in` property given with the access token object instead of just a magic 24h. I also wrote functions to GET the user and the user's guilds using this access token. Everything else remains the same.

Once again, thank you so much for making this! I'm forever grateful.
